content/docs/06-integrations/00-byoos.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Bring Your Own OS (BYOOS)"
+metaTitle: "Bring your own OS (BYOOS)"
+metaDescription: "Learn how to use a custom Operating System with a cluster profile."
+hiddenFromNav: true
+type: "integration"
+category: ['operating system']
+logoUrl: "https://registry.dev.spectrocloud.com/v1/spectro-proxy/blobs/sha256:b6081bca439eeb01a8d43b3cb6895df4c088f80af978856ddc0da568e5c09365?type=image/png"
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Bring Your Own Operating System (BYOOS) 
+
+The [Bring Your Own Operating System (BYOOS)](/https://docs.spectrocloud.com/cluster-profiles/byoos) enables the ability for you to use a custom Operating System (OS) with Palette. Palette comes with several OS out-of-the-box, but for some users, the existing list of OS may not meet their needs. 
+
+Using your custom OS provides several benefits, including the ability to control your own dependencies, improve performance, and ensure compatibility with your existing applications. With BYOOS, you can choose the OS that best fits your needs, whether it's a commercial or open-source distribution, and integrate it with your Kubernetes clusters. The BYOOS pack can be used with both Edge and non-Edge environments. 
+
+# Versions Supported
+
+**1.0.x**
+
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Edge" key="edge">
+
+## Prerequisites 
+
+To use the Edge BYOOS pack, you must have the following:
+
+<br />
+
+- The Edge Provider images are created and uploaded to a container registry. Refer to the [Bring Your Own OS](/clusters/edge/edgeforge-workflow/build-kairos-os) guide to learn how to create a custom OS. Refer to the [Build Images](/clusters/edge/edgeforge-workflow/build-images) guide for steps on how to create the Edge artifacts and how to upload your custom OS to a registry.
```

Comment:
```suggestion
- Edge Provider images you have created and uploaded to a container registry. Refer to the [Bring Your Own OS](/clusters/edge/edgeforge-workflow/build-kairos-os) guide to learn how to create a custom OS. Refer to the [Build Images](/clusters/edge/edgeforge-workflow/build-images) guide for steps on how to create the Edge artifacts and how to upload your custom OS to a registry.
```

Reply:
Do we need to say "Edge Provider OS images you have created and uploaded ... " ?